cpp/include/cugraph/mtmg/resource_manager.hpp

```diff
@@ -70,6 +70,8 @@ class resource_manager_t {
   {
     std::lock_guard<std::mutex> lock(lock_);
 
+    CUGRAPH_EXPECTS(remote_rank_set_.find(rank) == remote_rank_set_.end(),
```

Comment:
Just to double check, is `rank` still local? I assume `remote_rank_set_` stores global ranks, and can we query `remote_rank_set_` with a local rank?

Reply:
Or from now on, `rank` is global and `device_id` might serve as local GPU rank?

Reply:
In my implementation here (trying to somewhat mimic MPI) `rank` is global.  `device_id` is local.

Reply:
Yeah.. in that case, we need to review the documentation.
```
  /**
   * @brief add a local GPU to the resource manager.
   *
   * @param rank       The rank to assign to the local GPU
   * @param device_id  The device_id corresponding to this rank
   */
```
It is not clear to figure out that `rank` is global and `device_id` is local just seeing the documentation.

Reply:
Updated the parameter names and documentation to be `global_rank` and `local_device_id`.  Added a small improvement in the class documentation to identify the global rank concept.